Gemfile

```diff
@@ -15,9 +15,10 @@ gem 'virtus'
 gem 'rails-api'
 gem 'figaro'
 gem 'pg'
-
 gem 'active_model_serializers', '~> 0.10.0'
-
+gem 'will_paginate'
+gem 'responders'
```

Comment:
it's required for

https://github.com/department-of-veterans-affairs/vets-api/blob/prescription-api/app/controllers/rx_controller.rb#L13

and 

https://github.com/department-of-veterans-affairs/vets-api/blob/prescription-api/app/controllers/rx/v1/prescriptions_controller.rb#L20-L27